### PR TITLE
Return Cinema Details

### PR DESCRIPTION
**Pull request by jimbo8098**
Resolves issue https://github.com/seanmtracey/CineList-API/issues/20#issue-284828045

Call is made to /get/cinema/:venueID and returns:

```
{   
    name: 'Odeon Glasgow Quay - ODEON Luxe',
     address1: 'Glasgow',
     towncity: 'Glasgow',
     show_towncity: false,
     postcode: 'G5 8NP',
     website: 'http://',
     phone: '',
     distance: '-1',
     lat: '55.847691',
     lon: '-4.252370' 
}
```

This has the postcode *and* lat/long data which is exactly what the user was looking for. Other users may find the other information here useful.

### CODE REVIEW

**Comment by seanmtracey** [comment]
Cracking! Thanks :D

Looks good at first glance, will sit down this evening and pour over it properly before merging. Will let you know when it goes live.

**Comment by jimbo8098** [comment]
Perfect, I hope you like it!

**Comment by seanmtracey** [comment]
Out of interest, how did you find the new endpoint? I went hunting for one a little while ago and came up with nothing 😅

**Comment by jimbo8098** [comment]
Hehe when you query the cinemas, the postcode and lat long is there but you
need to specifically query each cinema from memory

**Comment by jimbo8098** [comment]
I just noticed my tabstops were set to 2 in Vim while I was editing. I'll correct that later if you like?

**Comment by seanmtracey** [comment]
Sure thing, but I was just planning on merging and correcting the code style later (nothing wrong with how you've done it, I just like to keep things consistent).

**Comment by jimbo8098** [comment]
Cool! I'll update the docs later tonight too, I just wanted to make sure it
was ok technically before I proceeded with that :)

**Comment by jimbo8098** [comment]
Fair play, I hadn’t even noticed it was wrong tabspace setting until I merged and checked the diff on Github!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

File: routes/get.js
Comment (seanmtracey) [inline routes/get.js]: We don't need the `return` here. Could you remove it when you get a mo'?

File: bin/lib/fafScraper.js
Comment (seanmtracey) [inline bin/lib/fafScraper.js]: Can we make the key for the cache a little less generic? Maybe `cinemaInfo`? I'd prefer that we make variables and keys as verbose as possible so that developers who are reading through it in the future don't have to use up to much headspace remembering what variables are being used where.

File: bin/lib/fafScraper.js
Comment (seanmtracey) [inline bin/lib/fafScraper.js]: ` data"),err;;` The `,err` is the wrong side of the `)` and we have one too many `;`

**Review by seanmtracey — CHANGES_REQUESTED** [review]
Minor stylistic changes and one error. Other than that LGTM. Have run locally (after merging with local master branch) and there are no conflicts.

Will merge once comments have been addressed :)

**Comment by seanmtracey** [comment]
Turns out I can make these changes myself in the PR! Neat!

Merged :)

**Pull request merged**
